src/app/services/vm-size.service.ts

```diff
@@ -49,34 +49,35 @@ export class VmSizeService {
     };
 
     private _sizes = new BehaviorSubject<List<VmSize>>(null);
-    private _excludedSizes = new BehaviorSubject<ExcludedSizes>(null);
+    private _includedSizes = new BehaviorSubject<IncludedSizes>(null);
```

Comment:
change type to `BehaviorSubject<IncludedSizes|null>`

do that for the others.  See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html

I enabled for the node environment but want to do everywhere so better start following the right format